src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (value1) => (value2) => {
+    count++;
+
+    const isPair = count % 2 === 0;
+
+    const result = count > 3 && isPair
+      ? 'Bzzz... Error!'
+      : value1 + value2;
```

Comment:
i don't think that is the best practice  to use ternary operators here, but it is subjectively

Reply:
ternary operator should be used about assignment values, right?